fee_allocator/fees_collected/v3_fees_2025-11-20_2025-12-04.json

```
@@ -0,0 +1,9 @@
+{
+  "optimism": 215508895,
+  "mainnet": 17729846623,
+  "avalanche": 1517452699,
```

this is also off? you write it correctly in your comment. not sure what you did here

```suggestion
  "avalanche": 1301943804,
```